integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/SlackSteps.java

```diff
@@ -31,4 +32,15 @@ public void slackActionTest(String messageText, int timeoutMinutes) {
                         () -> assertThat(SlackResource.getListOfSlackMessages()).as("Searching for message containing text: '%s'", messageTextWithoutPlaceholders)
                                 .anyMatch(msg -> msg.contains(messageTextWithoutPlaceholders)));
     }
+
+    @And("^create message with text \"([^\"]*)\" on slack channel$")
+    public void createMessageOnSlackChannel(String messageText) {
+        String messageTextWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, messageText);
+        Awaitility.await()
+                .atMost(Duration.ofMinutes(1))
+                .pollInterval(Duration.ofSeconds(1))
+                .untilAsserted(
+                        () -> assertThat(SlackResource.postToSlackWebhookUrl(messageTextWithoutPlaceholders)).isEqualTo(200));
```

Comment:
Why is this step implemented as a loop? Does the Slack POST request fail occasionally?

Reply:
I think just a consequence of copy and paste during development, will remove it